### PR TITLE
FOR REVIEW: Voted Operational Process Framework RPIP

### PR DESCRIPTION
**Pull request by LongForWisdom**
Will add discussions-to very shortly.

Needs general draft-level checks.

Hold off on doing final-status checks until I can get feedback from community.

### CODE REVIEW

File: RPIPs/RPIP-34.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+> The VOP-GMC-MEMBERSELECT definition describes how GMC elections take place.
+
+* A VOP definition contains the elements that make up a particular type of VOP.
+* A VOP definition MUST include the elements defined under the VOP Elements heading of this specification.
+* A VOP definition SHOULD follow the template defined under the VOP Definition Template heading of this specification.
+* If the use of the VOP framework is appropriate, a VOP definition MUST be included in the RPIP in which the process is described.
+
+
+#### VOP Elements
+* Code - A short, unique, human-readable string identifier that can be used to identify instances of this VOP. Should take the rough form of VOP-SUBJECT-ACTION
```
Comment (Valdorff) [inline RPIPs/RPIP-34.md:73]: Maybe "ID" instead of Code? Code is overloaded. Also works well with the "ID number" nomenclature used later. It's ok as is, fwiw, I just think clearer the other way.

File: RPIPs/RPIP-34.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+* Name - A longer identifier in plain English.
```
Comment (Valdorff) [inline RPIPs/RPIP-34.md:74]: Specific length? "in a few words" "in a handful of words"? open ended is fine if the intent is "user judgement"

File: RPIPs/RPIP-34.md [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+  * A VOP Record MUST include the date the VOP instance concluded.
+  * A VOP Record MUST include an appropriate URL link indicating the outcome of the VOP.
+  * A VOP Record MUST include a brief, human-readable summary of the outcome.
+
+---
+
+### VOP Usage - Overview
+> A Living informational RPIP that lists the currently defined VOPs and the RPIPs in which they are defined.
+
+* The VOP overview is a summary of defined VOPs within Rocket Pool.
```
Comment (Valdorff) [inline RPIPs/RPIP-34.md:125]: Can VOPs be undefined/removed/obsoleted?

File: RPIPs/RPIP-34.md [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+  * 
+  * 
+  * 
+* **Other Requirements:**   
+  * 
+  *
+  *
+```
+
+##### VOP-EXAMPLE-EXAMPLE
```
Comment (Valdorff) [inline RPIPs/RPIP-34.md:164]: I would suggest using `VOP-GMC-MEMBERSELECT-1` as your ongoing concrete example, or using an abstract example above. I think the mix is not great. Could do Example Management Committee (EMC) if desired as an in betweener.

File: RPIPs/RPIP-34.md [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+* **Purpose:** To allow the pDAO to override the GMC if it disagrees with a given grant decision.
+* **Responsible Party:** Rocket Pool Community
+* **Voting Group:** pDAO
+* **Vote Type:** Single Choice Plurality (Snapshot)
+* **Trigger(s):** Grant Announcement (within 2 weeks of announcement)
+* **Other Requirements:** See RPIP-26 - Expanded GMC Administrator Role for full requirements.
+
+Optional:
+* pDAO charter changes
+* oDAO charter changes
```
Comment (Valdorff) [inline RPIPs/RPIP-34.md:350]: maybe the odao responsibilities bits -- the rest of both charters are currently the _most_ difficult things in RP to change, so I definitely don't think they would use the VOP process

File: RPIPs/RPIP-34.md [hunk 6]
```diff
@@ -0,0 +1,365 @@
...
+* pDAO Budget Split
```
Comment (Valdorff) [inline RPIPs/RPIP-34.md:351]: pDAO direct expenses should be a VOP imo

File: RPIPs/RPIP-34.md [hunk 7]
```diff
@@ -0,0 +1,365 @@
...
+
+The inclusion of 'Other Requirements' does threaten to remove the value of standardization here. If every VOP is defining different other requirements. This is probably something to watch out for but it doesn't feel like you can just remove this field. At the very least, existing processes define more detailed requirements than can be captured in the current definition keys so this element needs to exist, at least at the point of transition. 
+
+### Records
+
+Records in some form are valuable. Tables make the most sense, as they are fairly compact, and make good use of horizontal space once you have a reasonable amount of columns.
+
+## Backwards Compatibility
+
+Upon ratification of this RPIP, existing ratified RPIPs containing processes that would fall under the VOP framework will be modified to use the VOP framework. The explicit intention is to maintain the logic and rules of existing processes, and just change their presentation such that they integrate with the VOP framework.
```
Comment (Valdorff) [inline RPIPs/RPIP-34.md:246]: This is a pretty significant thing that will happen - I'd like to see it in the specification.

In general, I think the specification should likely be leaner - some of the examples/templates could probably be in an implementation section.

**Review by Valdorff — COMMENTED** [review]
Some minor stuff - none of it big enough to block merging as Draft if the author desires

File: RPIPs/RPIP-34.md (hunk 3 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:125]: FML, Patrick flagged this to be yesterday and I forgot. TL;DR, yes. But then ideally you still want the records somewhere, which is somewhat annoying.

File: RPIPs/RPIP-34.md (hunk 5 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:350]: Yeah, more or less agree, see discussion thread.

File: RPIPs/RPIP-34.md (hunk 7 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:246]: Fair.

**Comment by LongForWisdom** [comment]
> Some minor stuff - none of it big enough to block merging as Draft if the author desires

Yes please. Merge as draft and will address the above prior to Finalization.

**Review by Valdorff — APPROVED** [review]
No Draft level blockers

File: RPIPs/RPIP-34.md (hunk 1 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:73]: Implemented in working copy.

File: RPIPs/RPIP-34.md (hunk 2 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:74]: Implemented in working copy.

File: RPIPs/RPIP-34.md (hunk 7 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:246]: Implemented in working copy.

File: RPIPs/RPIP-34.md (hunk 4 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:164]: Agree. Implemented in working copy.

File: RPIPs/RPIP-34.md (hunk 3 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-34.md:125]: Now covered removal and modification.
